workers/gitlab_issues_worker/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
at a later time, @ccarterlandis , this should be the SPDX license identifier and not the entire MIT License. Also, its 2020.